src/app/globals.css

```diff
@@ -25,3 +25,28 @@ body {
     )
     rgb(var(--background-start-rgb));
 }
+
+h1 {
+  font-size: 2em;
+  margin-bottom: 1rem;
+}
+
+h2 {
+  font-size: 1.5em;
+  margin-bottom: 0.5rem;
+}
+
+p {
+  margin-bottom: 0.5rem;
+}
+
+.centerText {
+  text-align: center;
+}
+
+.page-landing-background {
+  background-image: url("/ssd-background.jpg");
+  width: 100vw;
+  height: 100vh;
+  background-color: #cccccc;
```

Comment:
[Please Change]: The background image negates the need for a background color here.